error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssortedRules.java

```diff
@@ -115,6 +115,7 @@ T after(Iterator<T> iterator, T defaultValue) {
   // intelligently.
   static final class LogicalImplication {
     @BeforeTemplate
+    @SuppressWarnings("java:S2589" /* This violation will be rewritten. */)
```

Comment:
Note to self: perhaps we can come up with a better sentence for these cases.

Reply:
Maybe we can use the word "solve" in there. Explaining it will literally be solved the violation. "This code resolves the violation." 

Not happy with it per se, but just food for thought.

Reply:
Maybe `/* Application of this rule resolves the violation elsewhere. */`? :shrug: